webrtc.html

```diff
@@ -1880,6 +1880,16 @@ <h4>
                               <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SenderTransport]]}}.
                             </p>
                           </li>
+                          <li class="add-to-set-last-stable-offer-properties">
+                            <p>
+                              Set
+                              <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[LastStableRidlessSendEncodings]]}}
+                              to
+                              <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}} if
+                              <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}.length
+                              is `1` and the lone encoding [=map/contains=] no {{RTCRtpCodingParameters/rid}} member, otherwise `null`.
```

Comment:
```suggestion
                              If
                              <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}}.length
                              is `1` and the lone encoding [=map/contains=] no {{RTCRtpCodingParameters/rid}} member,
                              then set
                              <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[LastStableRidlessSendEncodings]]}}
                              to
                              <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[SendEncodings]]}};
                              Otherwise, set
                              <var>transceiver</var>.{{RTCRtpTransceiver/[[Sender]]}}.{{RTCRtpSender/[[LastStableRidlessSendEncodings]]}}
                              to `null`.
```